src/data_structures/src/bit_array.cairo

```diff
@@ -503,3 +498,34 @@ fn select(word: felt252, byte_index: usize, bit_index: usize) -> bool {
     };
     (shifted_bytes / shift_bit(bit_index).into()) % SELECT_BIT == 1
 }
+
+// TODO Remove all done, it was copied from Starkware's library. There must be a better way
```

Comment:
hein ?

Reply:
Was copied from the lib, maybe reworking the logic using their fn makes more sense, would leave in another PR

Reply:
fine, plz open minimal issue

Reply:
https://github.com/argentlabs/argent-contracts-starknet-private/issues/372

Reply:
https://github.com/keep-starknet-strange/alexandria/issues/291